clinet/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
The name of this dir should be `client`, not `clinet` :)

Reply:
Oops!
Sorry for that.